### ISSUE TITLE
Complete minimum example for webpack

### ISSUE THREAD

**Issue by dakhu** [roylee0704/react-flexbox-grid#83]
I'm new to react and webpack and having a hard time setting up this library (I'm not using webpack for my project, but have to set it up because it is a dependency of this lib). Readme mentions webpack config should have `include: /flexboxgrid/,` but `webpack.config.js` in this [example](https://github.com/roylee0704/react-flexbox-grid-example/blob/master/webpack.config.js) has no reference to `flexboxgrid`. Can you please point me to the right direction? Thanks

**Comment by curio77** [comment]
Yup, the example is outdated. Would appreciate an updated one as well!
